Web Development/README.md

```diff
@@ -172,6 +172,16 @@ this course has two sections:
     <td><a href="https://communitytaught.org/class/all">100devs</a></td>
     <td>100Devs is a diverse and inclusive community of 1000s of participants in a training program run by Leon Noel. It is 100% free and requires absolutely no previous technical experience. This is not a coding bootcamp, this is a how-to-get Full Stack Web Dev job bootcamp</td>
   </tr>
+
+  <tr>
+    <td><a href="https://part-time.learnhowtoprogram.com/tracks">The Odin Project</td>
```

Comment:
The url you provided seems not align with The Odin Project. Double check it and change it to something like this

Reply:
I will merged once you completed this issue. Thanks :)